source/FreeRTOS_Stream_Buffer.c

```diff
@@ -226,6 +226,9 @@ size_t uxStreamBufferGetPtr( StreamBuffer_t * const pxBuffer,
     const size_t uxNextTail = pxBuffer->uxTail;
     const size_t uxSize = uxStreamBufferGetSize( pxBuffer );
 
+    /* MISRA Ref 18.4.1 [The validity of values received from external sources]. */
+    /* More details at: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md#rule-184. */
+    /* coverity[misra_c_2012_rule_18_4_violation] */
```

Comment:
The message inside 18.4.1 [] seems to be from MISRA 4.14. Can we somehow avoid taking 4.14 exceptions? By doing basic NULL check.

Reply:
Updated the comment, the exception is indeed 18.4